book_config.py

```diff
@@ -130,75 +111,43 @@ def __init__(self, db, ids, parent):
     @property
     def book(self):
         return self._book_settings[self._index]
-    
-    def edit_asin(self, val):
-        self.book.asin = val
 
-    def edit_shelfari_url(self, val):
-        http_string = 'http://www.'
+    def edit_goodreads_url(self, val):
```

Comment:
I was trying to force the https part since our HTTPSConnection only accepts https pages. I figured I would add the www.goodreads.com part so that people don't try to enter in pages that aren't goodreads (in case they don't know about shelfari being gone or something). You're right though. This needs some work.